adr/0016-compliance-in-pepr.md

```diff
@@ -0,0 +1,51 @@
+# 14. Compliance in Pepr 
+
+Date: 2025-02-13
+
+
+## Status
+
+Pending
+
+
+## Context
```

Comment:
I get the feeling there is some context missing here with regards to the primary functions that pepr provides and the performance of compliance objectives. They are not innately the same and the _need_ to generate reports for compliance feels like it is missing more background. 

What does Pepr do today? What could pepr do to enable compliance? 

I think it is more than reporting - Evidence Collection -> Evaluation -> Aggregation -> Storage -> Reporting

Reply:
We're getting the concept rolling over in [compliance-reporter](https://github.com/defenseunicorns/compliance-reporter). Specifics around how & what compliance is with respect to Pepr will be iterated upon over there.